From 1d147f41031d0fb68a4658f6a1a2be9b9c37d0c6 Mon Sep 17 00:00:00 2001
From: Semih Yavuz <semih.yavuz@qt.io>
Date: Tue, 16 Sep 2025 13:13:56 +0200
Subject: [PATCH 038/239] Add security header for  src/qmlformat

There is no security critic code in qmlformat. Everything remains
at the default "signicant" security level.

QUIP: 23
Fixes: QTBUG-136198
Pick-to: 6.9 6.8
Change-Id: Iac1c84748bc98a9c921fc8ae895a2dcd53d01270
Reviewed-by: Ulf Hermann <ulf.hermann@qt.io>
(cherry picked from commit b855223eda6c124a08eea50163cc475b552da7f1)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/qmlformat/qqmlformatoptions.cpp  | 1 +
 src/qmlformat/qqmlformatoptions_p.h  | 1 +
 src/qmlformat/qqmlformatsettings.cpp | 1 +
 src/qmlformat/qqmlformatsettings_p.h | 1 +
 tools/qmlformat/qmlformat.cpp        | 1 +
 5 files changed, 5 insertions(+)

diff --git a/src/qmlformat/qqmlformatoptions.cpp b/src/qmlformat/qqmlformatoptions.cpp
index f8aa9c7f19..66c9383fdb 100644
--- a/src/qmlformat/qqmlformatoptions.cpp
+++ b/src/qmlformat/qqmlformatoptions.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2023 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant
 
 #include "qqmlformatoptions_p.h"
 #include "qqmlformatsettings_p.h"
diff --git a/src/qmlformat/qqmlformatoptions_p.h b/src/qmlformat/qqmlformatoptions_p.h
index 011ad4f9fa..3577bd3787 100644
--- a/src/qmlformat/qqmlformatoptions_p.h
+++ b/src/qmlformat/qqmlformatoptions_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2023 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant
 
 #ifndef QQMLFORMATOPTIONS_P_H
 #define QQMLFORMATOPTIONS_P_H
diff --git a/src/qmlformat/qqmlformatsettings.cpp b/src/qmlformat/qqmlformatsettings.cpp
index 181a3d9514..fb34fae41c 100644
--- a/src/qmlformat/qqmlformatsettings.cpp
+++ b/src/qmlformat/qqmlformatsettings.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2023 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant
 
 #include "qqmlformatsettings_p.h"
 
diff --git a/src/qmlformat/qqmlformatsettings_p.h b/src/qmlformat/qqmlformatsettings_p.h
index 9630439858..cc3c023603 100644
--- a/src/qmlformat/qqmlformatsettings_p.h
+++ b/src/qmlformat/qqmlformatsettings_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2023 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant
 
 #ifndef QQMLFORMATSETTINGS_P_H
 #define QQMLFORMATSETTINGS_P_H
diff --git a/tools/qmlformat/qmlformat.cpp b/tools/qmlformat/qmlformat.cpp
index 6cdfa907c3..8503af8447 100644
--- a/tools/qmlformat/qmlformat.cpp
+++ b/tools/qmlformat/qmlformat.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2019 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
+// Qt-Security score:significant
 
 #include <QCoreApplication>
 #include <QFile>
-- 
2.51.2

